src/export/bops/index.ts

```diff
@@ -348,25 +349,16 @@ export function getBOPSParams({
     });
 
   // 3. constraints
-  const constraints = (
-    passport.data?.["property.constraints.planning"] || []
-  ).reduce((acc: Record<string, boolean>, curr: string) => {
-    acc[curr] = true;
-    return acc;
-  }, {});
-  if (Object.values(constraints).map(Boolean).length > 0) {
+  if (passport.data?.["_constraints"]) {
+    const constraints: Record<string, boolean> = {};
```

Comment:
nit: Maybe `BOPSFullPayload["constraints"]` is a bit safer/cleaner?

Reply:
yep totally, thanks for spotting - updated now